### ISSUE TITLE
Good convergence, but bad performance even on training data.

### ISSUE THREAD

**Issue by Alien1007** [xingyizhou/CenterNet#528]
I have trained CenterNet with Hourglass for 140 epochs on my dataset including both training set and validation set. It shows an excellent convergence but I get a bad mAP.
Specifically, when I trained this model with DLA-34 on training set, the loss is about 0.7 when the training finished and it got about 63 mAP on validation set. And when I trained this model with more powerful Hourglass on both training set and validation set, it have converged very well and the loss fell down to about 0.25. However, it got only 38 mAP even on validation set, which has already been used for training.
I don't know what it implies.
